src/app/services/mqtt/transport.service.ts

```diff
@@ -14,35 +14,35 @@ export enum TransportState {
 }
 
 /* Interface which MQ Transports must implement */
-export interface TransportService {
+export abstract class TransportService {
```

Comment:
In these cases, it's better practice to use `abstract` classes instead of `interface`.